Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM debian:stretch-slim
 
+# This script is downloaded from http://hg.nginx.org/pkg-oss/raw-file/default/build_module.sh
```

Comment:
this comment should be added to the the script file rather than in the Dockerfile